azuracast/liquidsoap_custom.liq

```diff
@@ -243,9 +243,18 @@ playlist_lahmaloudclouds = cue_cut(id="cue_playlist_lahmaloudclouds", playlist_l
 playlist_lahmacun_presents = playlist(id="playlist_lahmacun_presents",mime_type="audio/x-mpegurl",mode="randomize",reload_mode="watch","/var/azuracast/stations/lahmacun_radio/playlists/playlist_lahmacun_presents.m3u")
 playlist_lahmacun_presents = cue_cut(id="cue_playlist_lahmacun_presents", playlist_lahmacun_presents)
 
+#Special event
+playlist_lahma_kripta_support = playlist(id="playlist_lahma_kripta_support",mime_type="audio/x-mpegurl",mode="randomize",reload_mode="watch","/var/azuracast/stations/lahmacun_radio/playlists/playlist_lahma_kripta_support.m3u")
+playlist_lahma_kripta_support = cue_cut(id="cue_playlist_lahma_kripta_support", playlist_lahma_kripta_support)
+
 #External streams
 #Example - REPLACE URL AND RENAME SOURCE NAME AS NEEDED AND ADD IT TO SCHEDULE (SEE BELOW IN THE MAIN SWITCH OPERATOR)
-#external_cashmere = mksafe(buffer(buffer=5., input.http("https://cashmereradio.out.airtime.pro/cashmereradio_b")))
+external_tirkultura = mksafe(buffer(buffer=5., input.http("https://s3.radio.co/s216811754/listen")))
```

Comment:
Please remove the external playlists and only keep the sample (out commented) one.